L1Trigger/L1TCalorimeter/plugins/L1TCaloRCTToUpgradeConverter.cc

```diff
@@ -100,17 +100,16 @@ L1TCaloRCTToUpgradeConverter::produce(edm::Event& iEvent, const edm::EventSetup&
     ROOT::Math::LorentzVector<ROOT::Math::PxPyPzE4D<double> > p4(0,0,0,0);
 
 
-    // create new format
-    //CaloRegion region;
-    CaloRegion region(*&p4,
-			   0.,
-			   0.,
-			   (int) rgn->et(),
-			   (int) rgn->id().ieta(),
-			   (int) rgn->id().iphi(),
-			   hwQual,
-			   0,
-			   0);
+    // create new format  TODO: not filled with necessary information
```

Comment:
I am not sure where or how to properly fill this format.  The old format seems to lack any distinction between ET and HT.